modules/azure_aks/main.tf

```diff
@@ -1,4 +1,4 @@
-# Reference: https://www.terraform.io/docs/providers/azurerm/r/kubernetes_cluster.html
+# Reference: 
```

Comment:
Updated link: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster

Reply:
Redirects to the link

Reply:
What is `Redirects to the link` ? There is no link to reference and/or redirect too. That's why I supplied the link. It was removed from the code.

Reply:
oops.. missed that the previous link was removed entirely, fixed it!